client/js/components/statement/splitStatement/SideBar.vue

```diff
@@ -8,107 +8,201 @@
 </license>
 
 <template>
-  <div class="side-bar">
-    <dp-label
-      :text="Translator.trans('tags')"
-      for="searchSelect"
-      class="u-mb-0" />
-
-    <div
-      v-if="editingSegment && editingSegment.tags.length > 0"
-      class="u-mt-0_5 u-mb-0_5">
+  <div
+    ref="sideBar"
+    class="side-bar flex flex-col">
+
+    <!-- Selected Tags Section -->
+    <div class="relative px-2 pt-2">
+      <dp-label
+        :text="Translator.trans('tags')"
+        for="searchSelect"
+        class="u-mb-0" />
       <div
-        v-for="(tag, idx) in editingSegment.tags"
-        :key="`tag_${idx}`"
-        class="tag inline-flex font-size-small u-mr-0_25 u-mb-0_25">
-        {{ tag.tagName }}
-        <button
-          type="button"
-          class="tag__remove btn--blank o-link--default u-ml-0_25"
-          @click="removeTag(tag.id)">
-          <dp-icon
-            icon="close"
-            size="small" />
-        </button>
+        v-if="editingSegment && editingSegment.tags.length > 0"
+        class="flex flex-wrap gap-1 bg-white">
+        <div
+          v-for="(tag, idx) in editingSegment.tags"
+          :key="`tag_${idx}`"
+          :class="assignTagSizeClasses(tag,idx)">
+          <div
+            :class="[
+              'tag flex whitespace-nowrap overflow-hidden text-sm px-0.5 py-0.5',
+              isTagAppliedToSegment(tag.id) ? 'bg-gray-500': 'bg-green-400',
```

Comment:
In fact, the green background color was not meant to be static, it should fade into the default bg color for those tags. However for this to work with css alone, we will have to teach Tailwind something like this...

```
  theme: {
    extend: {
      keyframes: {
        'background-transition': {
          '0%': { backgroundColor: '#48bb78' }, // Tailwind green-500
          '100%': { backgroundColor: '#a0aec0' } // Tailwind gray-500
        }
      },
      animation: {
        'background-transition': 'background-transition 1s ease-in-out forwards'
      }
    }
  }
```
This, however, would be best done in demosplan-ui. I'm gonna look into it tomorrow.

Another question: right now you're using the default tailwind colors - i realize that is already possible as we're not yet configured tailwind with other colors. However i'd suggest not to do so, as those colors won't exist. If https://github.com/demos-europe/demosplan-ui/pull/932  is merged and released, `bg-status-complete` vs. `bg-status-neutral` could be used here.

Reply:
what we discussed yesterday, here are some changes: https://github.com/demos-europe/demosplan-core/pull/3433/commits/27a4a72bb25868854ea3902179a019ed80358ed6